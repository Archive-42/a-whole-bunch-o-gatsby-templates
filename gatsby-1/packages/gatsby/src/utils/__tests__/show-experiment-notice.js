import { createNoticeMessage } from "../show-experiment-notice"
import stripAnsi from "strip-ansi"

describe(`show-experiment-notice`, () => {
  it(`generates a message`, () => {
    expect(
      stripAnsi(
        createNoticeMessage([
          {
            noticeText: `hi`,
            umbrellaLink: `http://example.com`,
            experimentIdentifier: `The Flag`,
          },
        ])
      )
    ).toMatchSnapshot()
  })
})
