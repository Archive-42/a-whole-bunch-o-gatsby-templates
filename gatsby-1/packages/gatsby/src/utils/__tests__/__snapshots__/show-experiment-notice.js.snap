// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`show-experiment-notice generates a message 1`] = `
"
Hi from the Gatsby maintainers! Based on what we see in your site, these coming
features may help you. All of these can be enabled within gatsby-config.js via
flags (samples below)

The Flag (http://example.com), hi
"
`;
